src/types.ts

```diff
@@ -198,7 +194,7 @@ export type PackageAfterDetermineVersion = Package & {
   incrementedVersion: string;
 };
 
-export type PackageChangelog = Record<string, ChangelogMessage[]>;
+export type PackageChangelog = Record<string, Message[] | RelatedMessages[]>;
```

Comment:
shouldn't it be `(Message | RelatedMessage)[]`.

Also `RelatedMessages` should be singular

Reply:
good point, changed it